dockers/docker-dhcp-relay/base_image_files/monit_dhcprelay

```diff
@@ -0,0 +1,7 @@
+###############################################################################
+## Monit configuration for DHCP_Relay container
+## process list:
+##	dhcrealy
+###############################################################################
+check process dhcprelay matching "/usr/bin/dhcrelay"
+	if does not exist then alert
```

Comment:
Yes. Monit can only monitor the process according to their name in host os. I will figure out a solution to fix this issue. Also we need a mechanism to dynamically decide the types of device such as T0 or T1.

Reply:
Device role (T0, T1, etc.) is determined at runtime via configuration.

Reply:
Got it.

Reply:
Joe, Can we try to use jinjia to write a template for Monit config file just like the supervisord.conf.j2 for supervisord in container dhcp_realy?

Reply:
Possibly. However, this information is not available until after the device has been configured, so we would need a separate mechanism to generate the monit config file at runtime and restart monit (if required).

Reply:
As we discussed offline, for this PR, please do not add monit config files for the dhcp_relay, radv and teamd containers. Since these containers spawn their processes dynamically based on configuration, you can create a separate PR with a different solution for these containers.

Reply:
will delete the corresponding config lines for these 3 containers.

Reply:
The pmon container also contains optional processes. Please delete the pmon changes as well; we will need a dynamic solution for it, also (see my earlier comment below).

Reply:
Yes. Also for lldp container. As you commented, the process name lldpd: connected will be decided by the number of neighbors. I will also delete this process.

Reply:
What? I think you misunderstood regarding lldp. There should always be an lldpd process running, it's just that the command line text is dynamic. You need to determine a more generic string to match against, so that it will always find that process, no matter how many neighbors are present.

Reply:
Got it.